lib/config/optionsconfiguration.go

```diff
@@ -41,6 +41,7 @@ type OptionsConfiguration struct {
 	OverwriteRemoteDevNames bool     `xml:"overwriteRemoteDeviceNamesOnConnect" json:"overwriteRemoteDeviceNamesOnConnect" default:"false"`
 	TempIndexMinBlocks      int      `xml:"tempIndexMinBlocks" json:"tempIndexMinBlocks" default:"10"`
 	UnackedNotificationIDs  []string `xml:"unackedNotificationID" json:"unackedNotificationIDs"`
+	UnpauseOnStart          bool     `xml:"unpauseOnStart" json:"unpauseOnStart"`
```

Comment:
What's the intention for this config? It's essentially a bool you can set to have other bools changed in the config on startup?